### ISSUE TITLE
Give option to disable spell-check autocorrection when naming a file

### ISSUE THREAD

**Issue by Korkman** [files-community/Files#7895]
### What's the Problem?

When creating a file, the name is auto-corrected to the case written in dictionary, for example:

`mercedes.txt` becomes `Mercedes.txt`

### Solution/Idea

While spellchecking is useful, at least the auto-correction should be optional. Maybe make "Preferences" > "Language" a collapsible like "Preferences" > "Startup settings" and have two toggles for spellchecking and auto-correct? I think might be helpful for most people, so it should be on by default, keeping current behavior for them.

### Alternatives

Turning off auto-correction entirely is also an option.

### Priorities

| Capability | Priority |
| :---------- | :------- |
| Prevent auto-correct when naming new files | Must |
| Keep the functionality for existing users via preferences, enabled by default | Should |

### Files Version

2.1.2.0

### Windows Version

Windows 10 Pro 21H2

**Comment by Korkman** [comment]
I just realized this is a feature which can be toggled in Windows settings globally. I don't know if it is feasible to make it optional within Files, or just guide users to the settings (is it possible to deep link into the Windows settings?)

**Comment by yaichenbaum** [comment]
@Korkman I don't think there is a way to change it in Files but the option in Windows is off by default. Is it still an issue if you turn off the option in Windows?

**Comment by Korkman** [comment]
@yair100 For me it is solved.

**Comment by gave92** [comment]
Just wanting to highlight that in XAML it's also possible to disable spell check for specific TextBoxes, so this feature request could make sense.

**Comment by yaichenbaum** [comment]
> Just wanting to highlight that in XAML should also be possible to disable spell check for specific TextBoxes (IsSpellCheckEnabled=False), so this feature request could make sense.

@gave92 I'll keep that in mind 👍

**Comment by dw-0** [comment]
> I don't think there is a way to change it in Files but the option in Windows is off by default.

@yair100 
I just stumbled across this behavior and wondered what is causing this. I just saw that this option was enabled in my Windows settings, although i never enabled it knowingly myself. So not sure if "is off by default" still applies? In my case it seemed to be on by default.

Also i am not sure if users want to disable this setting globally in case they make use of those functionalities in other applications. Even with auto-correction and spell-checking enabled, you won't see this in the Windows own file explorer when creating new folders or files, it just takes what you enter, the way you enter it.

In my case i just disabled the settings you mentioned, i guess for me it's solved then too. This post is just an info :)
